src/extensions/default/bramble/stylesheets/consoleTheme.less

```diff
@@ -26,6 +26,32 @@
     white-space: pre-line;
 }
 
+#editor-console .console .lines {
+    background: none;
+    border: 0;
+}
+
+#editor-console .console .lines .line{
+    display: flex;
+}
+
+#editor-console .console .lines .line .linenumber {
+    width: 35px;
+    padding: 6px 0px;
+    font-size: 15px;
+    text-align: right;
+    display: inline-block;    
+    color: #CCCCCC;
+    font-family: "Menlo Regular", Consolas, Inconsolata, "Vera Sans", "Lucida Console", Courier, monospace, fixed;
```

Comment:
Seems like you're duplicating a bunch of stuff from above, and I wonder if we can move some of this to a parent's rules vs. doing it specifically here.